src/main/kotlin/app/revanced/patches/youtube/layout/theme/Themes.kt

```diff
@@ -0,0 +1,15 @@
+package app.revanced.patches.youtube.layout.theme
+
+enum class Themes(val applier: (String) -> String?) {
```

Comment:
To make it clearer where the class Themes belongs, make it a child class of the `theme` patch.  The usage would be `ThemePatch.Themes.Amoled`.

Reply:
Sure, will do

Reply:
Done: 937201a85cc92c8683ee730fad5f45279200b503